simplecpp.cpp

```diff
@@ -788,6 +788,35 @@ void simplecpp::TokenList::readfile(Stream &stream, const std::string &filename,
             while (stream.good() && ch != '\r' && ch != '\n') {
                 currentToken += ch;
                 ch = stream.readChar();
+                if(ch == '\\') {
```

Comment:
can't you just use `peakChar` instead, something like:
```
    if (ch == '\\' && std::strchr("\r\n", stream.peakChar())) {
        // continue comment on next line..
        currentToken += "\\" + stream.readChar();
        continue;
    }
```
I don't know if `good()` should be used here.
please check if we handle windows "\r\n" line endings also.

Reply:
or a flag..
```
     bool backslash = false;
     while (stream.good() && ((ch != '\r' && ch != '\n') || backslash) {
         currentToken += ch;
         backslash = (ch == '\\');
         ch = stream.readChar();
     }
```